devnet/tokens/ITHEUM-a61317/info.json

```diff
@@ -0,0 +1,14 @@
+{
+  "website": "https://itheum.io",
+  "description": "The ITHEUM token forms the foundation for a decentralized web3 data economy, enabling vibrant and personalized metaverse worlds where data is traded with shared value between data creators and data consumers.",
+  "social": {
+    "blog": "https://itheum.medium.com",
+    "twitter": "https://twitter.com/itheum",
+    "telegram": "https://t.me/itheum",
+    "discord": "https://itheum.io/discord",
+    "whitepaper": "https://itheum.io/whitepaper",
+    "coinmarketcap": "https://coinmarketcap.com/currencies/itheum",
+    "coingecko": "https://www.coingecko.com/en/coins/itheum"
+  },
+  "status": "active"
+}
```

Comment:
optional: empty line at the end of the file

Reply:
Fixed